pull.py

```diff
@@ -57,8 +57,40 @@
         'FDLv1.2',
         'FDLv1.3',
     ],
+    'FreeArt': [ # any version
+        'LAL-1.2',
+        'LAL-1.3',
+    ],
     'FreeBSDDL': ['FreeBSD'],  # unify (multi-tag)
-    # FIXME: still working through this
+    'NPL': [ #any version
+        'NPL-1.0',
+        'NPL-1.1',
+    ],
+    'OSL': [ # any version through 3.0
+        'OSL-1.0',
+        'OSL-1.1',
+        'OSL-2.0',
+        'OSL-2.1',
+        'OSL-3.0',
+    ],
+    'RPL': [ # any version - Note that FSF website does not state any version, but references version 1.3 in the URL.  It is assumed that it also covers version 1.1 and 1.5, but this should be verified with FSF.
+        'RPL-1.1',
+        'RPL-1.3',
+        'RPL-1.5',
+    ],
+    'Unicode': [ # any version
+        'Unicode-DFS-2015',
+        'Unicode-DFS-2016',
```

Comment:
Here is another case where the FSF doesn't say “any version”.  [The version they link][1] is from 2012 (based on the copyright) and differs from `Unicode-DFS-2015` at least in:

> the above copyright notice(s) and this permission notice appear with all copies of the Data Files or Software,

vs. [`Unicode-DFS-2015`'s][2]

> this copyright and permission notice appear with all copies of the Data Files or Software,

Until we get clarity on this, can we drop it from this PR (and the associated `Unicode-DFS-*` `IDENTIFIERS` entries) and come back to them in follow-up work?

[1]: https://directory.fsf.org/wiki/License:Unicode
[2]: https://github.com/spdx/license-list-XML/blob/e5da40e25becb0aa7626d3f62649d2387284a623/src/Unicode-DFS-2015.xml#L43-L45